<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PriceListTreeGUI</class>
 <widget class="QWidget" name="PriceListTreeGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vista elenco prezzi</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="currentPriceDataSetLabel">
       <property name="text">
        <string>N. Preventivo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="currentPriceDataSetSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="selectionMode">
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="removePushButton">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addPushButton">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addChildPushButton">
       <property name="text">
        <string>+▼</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importPIDBPushButton">
       <property name="text">
        <string>EPA ►</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
